tests/state/Areas/AreaTrees/actions/setActiveKey-test.js

```diff
@@ -0,0 +1,22 @@
+import actions from '../../../../../src/state/Areas/AreaTrees/actions';
+import testBatchRunner from '../../../helpers';
+import {commonActionTypesObj as actionTypes} from '../../../../constants';
+
+const tests = [
+	{
+		name: 'setActiveKey',
+		action: (actions, actionTypes) => {
+			return actions.setActiveKey('k1');
+		},
+		dispatchedActions: [{type: 'AREAS.AREA_TREES.SET_ACTIVE_KEY', key: 'k1'}],
+	},
+];
+
+const dataType = 'areaTree';
+const categoryPath = 'metadata';
+describe(
+	'state/Areas/AreaTrees/actions',
+	testBatchRunner(dataType, categoryPath, tests, actions, actionTypes)
+);
+
+export default tests;
```

Comment:
I'm confused. setActiveKey is part of a common test set, so why is it here specifically?

Reply:
I would like to answer something reasonable, but it was done at the end of the shift.  Fixed: d51baa0f7d34d195a68bdf89ab1a96bc693e7205